src/cmd/create.go

```diff
@@ -115,11 +115,9 @@ func create(cmd *cobra.Command, args []string) error {
 			return errors.New("this is not a toolbox container")
 		}
 
-		if _, err := utils.ForwardToHost(); err != nil {
-			return err
-		}
-
-		return nil
+		cmd.SilenceErrors = true
```

Comment:
I wonder if we can set `SilenceErrors` on the root command and take more responsibility for printing the errors.  That way we wouldn't need this boilerplate in each command.

Reply:
I have submitted https://github.com/containers/toolbox/pull/1533 to verify the idea.  Let's see if it works out.

Reply:
It seems to work.

Reply:
So, I resurrected this change based on setting `SilenceErrors` on the root command in https://github.com/containers/toolbox/pull/1534